tests/helpers/test_selector.py

```diff
@@ -191,7 +191,13 @@ def test_object_selector_schema(schema):
 
 @pytest.mark.parametrize(
     "schema",
-    ({}, {"multiline": True}, {"multiline": False}),
+    (
+        {},
+        {"multiline": True},
+        {"multiline": False},
+        {"placeholder": "hh:mm:ss"},
```

Comment:
You should use a time selector for this

Reply:
time selector can not be use use to select a time interval ... only an absolute  time in 24 hour format.
If you try to use as a time interval, to get to the second 12 hour period you must specify PM. 
Not very intitive. And could cause users to be confused between an absolute time and a time interval.  
Time selector should have had an option for the UI to be in 24 hour format.

This was just one example .. there are lots of use cases for structured text input where you hava a validation pattern and placeholder to remind the user of the required validation format.
i.e.
     phone number for alarm system 
     exactly four or six digits for door or alarm panel code
    ...

Reply:
We have UI for both time intervals and just time, so if that is the goal, I think it makes more sense to create a selector for that?